### PR TITLE
take google service version from environment if available

### PR DESCRIPTION
**Pull request by fattahmuhyiddeen**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -1,5 +1,10 @@
 apply plugin: 'com.android.library'
 
+def DEFAULT_COMPILE_SDK_VERSION             = 23
```
Comment (cbrevik) [inline android/build.gradle]: Could you add the other project properties as well? For `compileSdkVersion `, `buildToolsVersion` and `targetSdkVersion`

**Review by cbrevik — CHANGES_REQUESTED** [review]
Thanks for the PR! 🎉 Could you also add docs in `README.md` on how to use this?

**Comment by yangtaufoo** [comment]
Thanks @fattahmuhyiddeen & @cbrevik!

**Comment by fattahmuhyiddeen** [comment]
ok done put in README.md

**Review by cbrevik — APPROVED** [review]

**Comment by cbrevik** [comment]
Awesome, thanks for this. It'll be included in the next version :+1:

**Comment by cbrevik** [comment]
Published as `5.6.2`

**Comment by christocracy** [comment]
Here’s a wiki entry in my [react-native-background-geolocation](https://github.com/transistorsoft/react-native-background-geolocation/wiki/Solving-Android-Gradle-Conflicts) module which nicely explains usage.  You might borrow chunks to add to your README.

**Pull request merged**